client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -53,9 +51,16 @@ const generatePluginDescriptionWithLink = (
 	} );
 };
 
-const installableExtensions = [
+const primaryTitle = __( 'Get the basics', 'woocommerce-admin' );
```

Comment:
What do you think about offloading this to the remote specs as well?  If we plan to change this in the future, it might be nice to be able to allow the remote to create the categories.

```json
[
	{
		"key": "basics",
		"locales": [
			{
			"locale": "en_US",
			"title": "Get the basics",
			}
		],
		"plugins": [ ... ]
	}
]
```

Reply:
That would make things easier from a data handling perspective as well because its the same shape as the local data. The problem with the config from above is that properties with an array of rules don't get evaluated unless they are at the top level. In other words, `is_visible: [ <rules> ]` gets returned instead of `is_visible: true`.

I also agree that it would be nice to change the language on the fly. Ultimately, its not the primary goal of the issue so I kept that in the client.